pkg/services/ngalert/state/testing.go

```diff
@@ -13,25 +14,31 @@ var _ InstanceStore = &FakeInstanceStore{}
 
 type FakeInstanceStore struct {
 	mtx         sync.Mutex
-	RecordedOps []any
+	recordedOps []any
```

Comment:
Tests were racing to read this member field while a background goroutine from the runtime code being tested was still trying to write to it. This happens in the tests looking for an eventual state.